rules.d/80-net-setup-link.rules

```diff
@@ -3,6 +3,7 @@
 SUBSYSTEM!="net", GOTO="net_setup_link_end"
 
 IMPORT{builtin}="path_id"
+IMPORT{builtin}="net_driver"
```

Comment:
Ok, I have moved it to 50-udev-default.rules. It makes sense. One of the reasons I started to work on this was my surprise that it was not set earlier, so maybe this will spare others from the same.